### PR TITLE
diarrheal diseases: added restrictions table

### PR DESCRIPTION
**Pull request by beatrixkh**

### CODE REVIEW

File: docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst [hunk 1]
```diff
@@ -156,4 +156,35 @@ Data Descriptions
 	  - YLD appendix
 	  - Disability weight of sequela with id {id}
 	  -
-	
\ No newline at end of file
+
+.. list-table:: Restrictions
```
Comment (collijk) [inline docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst]: Shouldn't be in this PR.  If you're working on totally separate documents, remember to make all branches from master.

File: docs/source/models/gbd2017/causes/diarrheal_diseases/index.rst [hunk 2]
```diff
@@ -222,6 +222,37 @@ Data Description
...
+	  - age_group_id = 2; [0-0.01917808 years)
+	* - YLL age group end
+	  - 90-94
+	  - age_group_id = 32; [90-94 years)
+	* - YLD age group start
+	  - Early neonatal
+	  - age_group_id = 2; [0-0.01917808 years)
+	* - YLD age group end
+	  - 90-94
+	  - age_group_id = 32; [90-94 years)
```
Comment (NathanielBlairStahn) [inline docs/source/models/gbd2017/causes/diarrheal_diseases/index.rst]: GBD mapping says the age group id should be 235, and so should the one for yll age group end.

File: docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst (hunk 1 above)
Comment (beatrixkh) [reply docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst]: oops; will make sure to do so in the future. thanks!

File: docs/source/models/gbd2017/causes/diarrheal_diseases/index.rst (hunk 2 above)
Comment (beatrixkh) [reply docs/source/models/gbd2017/causes/diarrheal_diseases/index.rst]: thanks for the catch!

**Pull request merged**